### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by baoshan** [github/pjax#1]
__Should I stick with v1 or should I take part in the WIP:question:__

**Comment by hunt3r** [comment]
Yes, good question, we're evaluating some pjax implementations, we're looking at this one too: 
https://github.com/MoOx/pjax
